src/displayapp/DisplayApp.cpp

```diff
@@ -226,6 +226,10 @@ void DisplayApp::Refresh() {
               default:
                 break;
             }
+          } else if (currentApp == Apps::NotificationsPreview && gesture == TouchEvents::SwipeRight) {
+            auto currentId = notificationManager.GetLastNotification().id;
```

Comment:
Is it necessary to write the ID to a variable? 

Without testing my claim, `notificationManager.Dismiss(notificationManager.GetLastNotification().id);` would also work or am I missing something?

Reply:
You are right, I just think putting it into two lines makes it a little easier to read. In the end the compiler optimizes both versions to the same result I believe.